templates/event_list.html

```diff
@@ -1,10 +1,13 @@
 {% load bootstrap4 %}
 {% bootstrap_css %}
 
+{% load el_pagination_tags %}
+
 <h1>Events</h1>
-<div class="container-fluid">
+
+<div class="container-fluid" style="width:80%">
```

Comment:
You should create a file to put styles. Then you can import this file in the html file.  check this https://docs.djangoproject.com/en/3.0/intro/tutorial06/

Reply:
Thank you for the observation. I've just fixed it